docs/deployment.md

```diff
@@ -88,3 +88,23 @@ To deploy to production server:
   ```shell
   sudo systemctl start docker-osmaxx.service
   ```
+
+## Deploying a Hotfix
+
+Follow the above steps until 5a.
+
+Replace step 5b with:
+
+5. 
```

Comment:
Markdown will convert this to a numbered list, starting at **1.** rather than **5.**.